tests/Controller/CitizenActionControllerTest.php

```diff
@@ -108,7 +168,18 @@ public function testUnregistrationFailed(): void
         $this->client->request(Request::METHOD_GET, sprintf('/action-citoyenne/%s', $citizenAction->getSlug()));
 
         $this->assertSame('S\'inscrire', $this->client->getCrawler()->filter('a.newbtn--orange')->text());
-        $this->assertNull($this->getEventRegistrationRepository()->findAdherentRegistration($uuid, LoadAdherentData::ADHERENT_2_UUID));
+        $this->assertNull($this->repository->findAdherentRegistration($uuid, LoadAdherentData::ADHERENT_2_UUID));
+    }
+
+    private function seeMessageSuccesfullyCreatedFlash(Crawler $crawler, ?string $message = null)
```

Comment:
It's not the first time I see this method in a test we should move it to the `ControllerTestTrait` and rename it `seeFlashMessage`.
Also, when no message is passed, we should assert that `0 === count($flash)`.